src/proto/Polkadot.proto

```diff
@@ -68,11 +70,11 @@ message SigningInput {
     bytes genesis_hash = 2;
     uint64 nonce = 3;
     uint32 spec_version = 4;
-    bytes tip = 5;  // big integer
-    Era era = 6;    // empty means Immortal
-    bytes private_key = 7;
-    Network network = 8;
-    uint32 extrinsic_version = 9;
+    uint32 transaction_version = 5;
+    bytes tip = 6;  // big integer
+    Era era = 7;    // empty means Immortal
+    bytes private_key = 8;
+    Network network = 9;
```

Comment:
should this be just int value? I assume we can get that info from rpc?

Reply:
this is also mapped to substrate address type (polkadot/kusama and other substrate based chain), we need it to initialize correct address